elements/pfe-switch/BaseSwitch.ts

```diff
@@ -84,6 +84,10 @@ export abstract class BaseSwitch extends LitElement {
   }
 
   #toggle() {
+    if (this.disabled) {
```

Comment:
this is woke
but bespoke? put `pointer-events: none` on the disabled host. use

```css
:host([disabled]),
:host(:disabled) {
  pointer-events: none;
}
```

Reply:
Are you suggesting that I remove the early return in favor of the CSS change? Or are you suggesting that I add this too?

Reply:
i think the css would do the trick on it's own, but i don't object to the early return

Reply:
make sure to put it in base styles since it's not pfe-specific